tests/erp/mapper/test_gendarmerie.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+
+from erp.import_datasets.fetcher_strategy import StringFetcher
+from erp.models import Activite, Erp
+from tests.erp.mapper.fixtures import gendarmeries_valid
+from erp.import_datasets.import_datasets import ImportDatasets
+from erp.mapper.gendarmerie import RecordMapper
+
+
+@pytest.fixture
+def activite_cdv(db):
+    return Activite.objects.get_or_create(nom="Gendarmerie")
+
+
+@pytest.fixture
+def import_dataset(gendarmeries_valid, db, activite_cdv):
+    def _factory(dataset=gendarmeries_valid):
+        fetcher = StringFetcher(dataset, RecordMapper.fields)
+        mapper = RecordMapper(fetcher=fetcher, dataset_url="dummy")
+        return ImportDatasets(mapper=mapper, is_scheduler=True)
+
+    yield _factory
+
+
+def test_basic_stats(import_dataset, gendarmeries_valid):
+    imported, skipped, errors = import_dataset().job()
+    erp = Erp.objects.filter(
+        source_id=gendarmeries_valid[0]["identifiant_public_unite"]
+    ).first()
+    assert erp is not None
+    assert imported == 3, skipped == 0
```

Comment:
You can't chain multiple asserts that way:

```py
In [1]: assert True is True

In [2]: assert True is True, False is True

In [3]: assert False is True
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-3-826790e2fb9e> in <module>
----> 1 assert False is True
```

Reply:
Wow thanks, I tried to refactor my tests and didn't noticed that. This was too reckless.